### PR TITLE
Data tables

### LINKED ISSUE #9: Data Table / List
Should list the most important data of the EMPLOYEE model, and be sortable/filterable on those fields.

Here's an initial stab at the columns:
```
First Name
Last Name
Date of Birth
Email
Phone Number
Skills
```

### PR DESCRIPTION
**Pull request by arasevic**
Initially displaying employees name, email, phoneNumber and birthDate
Filters based on name
sorts on all columns
has pagination

Resolves #9

### CODE REVIEW

**Comment by brendoncaulkins** [comment]
Resolves #9

File: src/app/home/home.component.ts [hunk 1]
```diff
@@ -13,17 +14,57 @@ import { EmployeesService } from '../services/employees/employees.service'
...
         this.employees = []
       }
     })
+    this.dataSource = new MatTableDataSource(this.employees)
+  }
+  ngOnInit() {
+    this.dataSource.filterPredicate = (employee: IEmployee, filter: string) => {
+      console.log(employee)
+      console.log(filter)
+      return employee.bio.firstName.toLowerCase() == filter
```
Comment (brendoncaulkins) [inline src/app/home/home.component.ts]: This probably should be `return employee.bio.firstName.toLowerCase().includes(filter.toLowerCase())`

That way, it's totally case insensitive and works for sub-strings (jo yields John and Johanssen, or Joan, etc).

We may want to also extend it to last name as well...
`return employee.bio.firstName.toLowerCase().includes(filter.toLowerCase()) || employee.bio.lastName.toLowerCase().includes(filter.toLowerCase())`

Comment (brendoncaulkins) [reply src/app/home/home.component.ts]: I can't tag it directly, but the `unsubscribe` in `ngOnDestroy` needs parens.  `unsubscribe()`

Comment (arasevic) [reply src/app/home/home.component.ts]: fixed @brendoncaulkins good catches!

**Pull request merged**